index.html

```diff
@@ -227,8 +215,29 @@ <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
           </ul>
       </div>
     </section>
+    <section class="contact">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+    <form action="https://formspree.io/f/xgebvavj" method="POST" class="contact-form">
+      <input type="text" class="contact-input" placeholder="Full name" maxlength="30" required>
+      <input type="text" class="contact-input" placeholder="Email address" required>
+      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500">Enter text here</textarea>
```

Comment:
Very cool `textarea`, I love it 😻 but instead of adding a predetermined text to it (user have to erase before writing their own message), you could consider using the `placeholder` attribute. Your code would look like this then:
```
      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Enter text here"></textarea>
```